orangecontrib/geo/cc_cities.py

```diff
@@ -5,11 +5,10 @@
 _CC_NAME_TO_CC_NAME = {
     # EU
     'Kosovo': ['Kosovo, Republic of'],
-    'Northern Cyprus': ['N. Cyprus', 'North Cyprus'],
-    'Aland': ['Åland Islands', 'Åland'],
-    'Bosnia and Herzegovina': ['Bosnia and Herz.'],
-    'Czech Republic': ['Czech Rep.'],
-    'Faroe Islands': ['Faeroe Is.'],
+    'Åland': ['Åland Islands', 'Aland'],
+    'Bosnia and Herz.': ['Bosnia and Herzegovina'],
+    'Czechia': ['Czech Rep.', 'Czech Republic'],
+    'Faeroe Is.': ['Faroe Islands', 'Faeroe Islands'],
```

Comment:
The mapping is [inverted later on](https://github.com/biolab/orange3-geo/pull/47/files#diff-e25d8f6b38b4b06194c1187a4380ea00R342). Left are the names used in GeoJSON files, right are synonyms. So some of these were in fact wrong?

Reply:
For instance:
```bash
$ cd Orange3-geo/orangecontrib/geo/geojson
$ grep 'Lao PDF' *.json
admin0.json: ... "properties": {"name": "Lao PDR", "adm0_a3": "LAO", "_id": "LAO", "iso_a3": "LAO", "iso_a2": "LA", "sovereignt": "Laos", "type": "Sovereign country", "subregion": "South-Eastern Asia", "economy": "7. Least developed region", "fips10": "LA", "region_wb": "East Asia & Pacific", "continent": "Asia", "region_un": "Asia"}} ...

$ grep 'Czechia' *.json  # No match
$
```

Reply:
I am aware of the invert. I am working with the json files I got from the json branch. I infect get matches when I run `grep 'Czechia' *.json`. I change mappings were the key was not in CC_SHAPES (from the code I assume it contains the geojson data).

Reply:
I regenerated the JSONs from sources and can no longer reproduce my above oservations. Your changes seem correct(er). :+1: